server/app.js

```diff
@@ -50,6 +52,6 @@ app.use(function(err, req, res, next) {
 
 //listening to this port 
 const PORT = process.env.PORT || 4000;
```

Comment:
you may want to change this to default 3001 because that's the default proxy in package.json

Reply:
or, vice versa, change the proxy in package.json